examples/regression/4-tutorials/plot_crf_tutorial.py

```diff
@@ -0,0 +1,274 @@
+"""
```

Comment:
Why do the tutorials show up in this order? I think the main regression tutorial should show up first?

<img width="726" alt="Screenshot 2023-08-31 at 10 48 47" src="https://github.com/scikit-learn-contrib/MAPIE/assets/63096108/bad3a076-09c8-41d7-9cca-aedcaca115e4">

Reply:
To be fixed in issue #348.